operators/tool_tracking_postprocessor/CMakeLists.txt

```diff
@@ -25,10 +26,21 @@ add_library(tool_tracking_postprocessor SHARED
   tool_tracking_postprocessor.cuh
   )
 
-set_target_properties(tool_tracking_postprocessor PROPERTIES CUDA_ARCHITECTURES "70;80")
+set_target_properties(tool_tracking_postprocessor
```

Comment:
I see CUDA_ARCHITECTURES was removed. Are these already being set at the top-level of the project?

Reply:
No, I don't think so. If it is not specified then it's using the default chosen by ncc. Using "70;80" seems to be somewhat artificially used. Using "all" like in HSDK does not work since the x86 CUDA driver does not include the kernels for mobile GPUs (e.g. SM87 (Orin)). Since the compiled code usually is used on the installed GPU using that option seems to make most sense.